### DISCUSSION TITLE
MP Handtracking does not shown when view patch

### DISCUSSION THREAD

**Discussion by dmarcst** [cables-gl/cables_docs#530] [! Welcome - readme first!]
I have Handtracking on my project, and it works fine when I view the file in editor mode. However, when I send out a secret link to preview, it seems to doesn't work anymore and only shows my background image. I've attached photos.

**In editor mode**
<img width="644" alt="Screenshot 2023-05-20 at 11 22 43" src="https://github.com/cables-gl/cables_docs/assets/119349517/eb542024-19ce-48bc-9932-2f9d26db7bc9">
**Secret link preview**
<img width="644" alt="Screenshot 2023-05-20 at 11 25 04" src="https://github.com/cables-gl/cables_docs/assets/119349517/6abf823f-7070-4b62-8dc4-8b8d62c8abde">

Thank you for your support!

**Comment by steam0r** [comment]
do you get any errors in the console? could you share public patch or secret link that shows the problem?
